### DISCUSSION TITLE
posts/lca-binary-lifting/

### DISCUSSION THREAD

**Discussion by giscus** [alikia2x/blog#1] [comments]
# posts/lca-binary-lifting/

暑假来临，寒寒参加OI集训，今日学到了如何使用倍增法查找一颗树的LCA（最近公共祖先），故在此分享。

https://blog.alikia2x.com/posts/lca-binary-lifting/

**Comment by alikia2x** [comment]
第一条评论～～
其实之前不太愿意给博客加评论的
但想了想，还是加上吧(｡･ω･｡)ﾉ
